Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -4160,6 +4160,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         new DiscretePropertyWrapper<Style::ViewTransitionName>(CSSPropertyViewTransitionName, &RenderStyle::viewTransitionName, &RenderStyle::setViewTransitionName),
         new DiscretePropertyWrapper<FieldSizing>(CSSPropertyFieldSizing, &RenderStyle::fieldSizing, &RenderStyle::setFieldSizing),
         new DiscretePropertyWrapper<const Vector<Style::ScopedName>&>(CSSPropertyAnchorName, &RenderStyle::anchorNames, &RenderStyle::setAnchorNames),
+        new DiscretePropertyWrapper<const NameScope&>(CSSPropertyAnchorScope, &RenderStyle::anchorScope, &RenderStyle::setAnchorScope),
```

Comment:
Can you add anchor-scope to https://searchfox.org/wubkat/rev/57a8e4876bcf0f8b4ce8d0bcec82b60e2fbc94b2/LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/property-list.js#28 to make sure this gets tested?

Reply:
That's WPT right? So once I've added it, I should export to WPT also?

Reply:
Can you check that I do it correctly?

Reply:
Yep you need to export, looks correct to me.

Reply:
Note that you might need platform specific rebaselines

Reply:
I'll wait for the tests to run then rebaseline if it fails on other platforms.